app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :post
+  validates :user, presence:true
+  validates :post, presence:true
```

Comment:
user and post are foreign keys here, they would automatically be filled in the record, validation isn't necessary